pkg/aws/awscontrolplane/validate_awscontrolplane.go

```diff
@@ -261,6 +315,28 @@ func contains(s []string, e string) bool {
 	}
 	return false
 }
+func countUniqueValues(s []string) int {
+	counter := make(map[string]int)
+	for _, a := range s {
+		counter[a]++
+	}
+	return len(counter)
+}
+func orderChanged(old []string, new []string) bool {
+	if len(old) > len(new) {
+		temp := old
+		old = new
+		new = temp
+	}
+	for i, o := range old {
+		for _, n := range new {
+			if o == n && o != new[i] {
```

Comment:
What we ensure here is that elements from the old object remain in their original position in the new object. However, it is okay for them to appear more times and also not at all.